docs/registries.conf.5.md

```diff
@@ -7,35 +7,44 @@ registries.conf - Syntax of System Registry Configuration File
 
 # DESCRIPTION
 The REGISTRIES configuration file is a system-wide configuration file for container image
-registries. The file format is TOML.
+registries. The file format is TOML.  The only valid categories are: 'registries.search',
+'registries.insecure', and 'registries.block'.
```

Comment:
I lifted this sentence from another conf file, but can't find it now.  The reason I added was to give the user a quick look at the sections and to act as a backup in case one or more were accidentally removed.  How's about I soften the sentence slightly and remove 'only'?